test/config/utility.cc

```diff
@@ -265,7 +263,7 @@ std::string ConfigHelper::discoveredClustersBootstrap(const std::string& api_typ
       grpc_services:
         envoy_grpc:
           cluster_name: my_cds_cluster
-      set_node_on_first_message_only: false
+      set_node_on_first_message_only: true
```

Comment:
If this is default false, why not leave the integration tests also default false and just have some dedicated tests for it?

Reply:
This is just trying to revert to how things were before #7293, #8478, and #9044. The first added the TODOs/switched true to false, the second removed the TODOs/switched false to true, and the third was a partial revert

Reply:
OK. Do we have any tests that verify flows when it is false?

Reply:
Looking at #7876, I don't think so, because `expect_node = false` by default

Reply:
Could you add at least one? Thanks.